tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc

```diff
@@ -91,6 +90,8 @@ int initWeightPrePackingCacheCapacity() {
 
 }  // namespace
 
+std::thread::id kDiscCpuDefaultThreadId{};
```

Comment:
kDiscCpuDefaultThreadId is not a const?

Reply:
Yes, it's not a const. We just want a process-level unique thread id.

Reply:
Then I think the naming should not start with `kName`

Reply:
Nit: rename `kDiscCpuDefaultThreadId` to `discCpuDefaultThreadId`

Reply:
I think c++ also supports runtime-initialized const? I say it's not a const above may not be accurate. The value of this object is indeed not able to change after initialized.

Reply:
Is the `const` specifier capable for the std::thread::id?